### PR TITLE
Regenerated from api - added BannedUser type

### PR DESCRIPTION
**Pull request by sbryzak**
Signed-off-by: Shane Bryzak <sbryzak@gmail.com>

### CODE REVIEW

File: deploy/olm-catalog/host-operator/host-operator.package.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+channels:
```
Comment (alexeykazakov) [inline deploy/olm-catalog/host-operator/host-operator.package.yaml]: There should not be any package yamls

Comment (sbryzak) [reply deploy/olm-catalog/host-operator/host-operator.package.yaml]: I've removed this yaml file, but I'd like to know why it is getting generated in the first place.  I've removed my vendor dir and run `make clean` in the api project, so I'm not sure where this is coming from.

Comment (MatousJobanek) [reply deploy/olm-catalog/host-operator/host-operator.package.yaml]: That's weird. Are you sure that you use the latest versions of both projects? What target do you use for generating and propagating the CRD changes in api project? You should use `make generate` - that should do everything necessary for you including creating/updating the correct CSV and operator package

**Comment by alexeykazakov** [comment]
> Please, don't forget to add also the BannedUser to the list of owned CRDs in the base template CSV: https://github.com/codeready-toolchain/host-operator/blob/master/deploy/olm-catalog/toolchain-host-operator/0.0.1/toolchain-host-operator.v0.0.1.clusterserviceversion.yaml#L115
you should also add an example of the CR here: https://github.com/codeready-toolchain/host-operator/tree/master/examples
When it's done, then you need to propagate the change also to the hack files - to do that you need to either run make generate-csv or directly by calling the script: ../api/scripts/olm-catalog-generate.sh -pr ../host-operator

I think we need to document it somewhere. Maybe in api makefile (generate target)?

**Comment by alexeykazakov** [comment]
> I think we need to document it somewhere. Maybe in api makefile (generate target)?

Or/and in Readme. There is a few places which need to be changed in case of new CRD introduction.

**Comment by MatousJobanek** [comment]
@sbryzak you didn't do all the steps that I wrote in the comment: https://github.com/codeready-toolchain/host-operator/pull/134#pullrequestreview-339892906 - talking about this one:

>When it's done, then you need to propagate the change also to the hack files - to do that you need to either run make generate-csv or directly by calling the script: ../api/scripts/olm-catalog-generate.sh -pr ../host-operator

without having it propagated in the hack files, you won't be able to verify the changes locally.

**Comment by MatousJobanek** [comment]
I created PR with the missing bits https://github.com/codeready-toolchain/host-operator/pull/135

**Pull request merged**